.nvmrc

```diff
@@ -0,0 +1 @@
+4.2.6
```

Comment:
The `.nvmrc` is helpful for developers who use `avn` on unix-like OS.  I like it because when I `cd` into this dir, the correct version of node is automatically configured for me.  So I recommend/prefer to have it so I don't have to manually type `nvm use` or `nvm use 6`.